DAQConditions/fcl/prolog.fcl

```diff
@@ -0,0 +1,17 @@
+BEGIN_PROLOG
+
+EventTiming : {
+   useDb : false
+   verbose : 0
+
+   # MHz
+   SystemClockSpeed : 40.0
+   # ns
+   TimeFromProtonsToDRMarker : 200.0
+   # clock counts
+   OffSpillEventLength : 4000
+   OnSpillBins : 5
+   OnSpillEventMaxLength : 68
+}
+
+END_PROLOG
```

Comment:
I support the idea of DAQConditions package, we need a place to keep system-wide parameters like this.  Do you think this should be home for subsystem-specific DAQ-related conditions?

Reply:
I'd say its fine if only the related parameters were here, I don't want to have to pull already existing things out of StrawElectronics for example.